src/tetris/field/remoteField.ts

```diff
@@ -2,24 +2,42 @@
 
 import Block from 'tetris/brick/block';
 import Vector2 = Phaser.Math.Vector2;
+import MatchPlayer from 'tetris/interfaces/MatchPlayer';
 import config from 'tetris/config';
 
 export default class RemoteField {
 	//region public members
+	get height(): number {
+		return this._height * config.field.blockSize * this._drawScale;
+	}
+
+	get width(): number {
+		return this._width * config.field.blockSize * this._drawScale;
+	}
+
+	get x(): number {
+		return this._drawOffset.x;
+	}
+
+	get y(): number {
+		return this._drawOffset.y;
+	}
+
+	set x(x: number) {
+        this._drawOffset.x = x;
+        this._adjustTextX();
+	}
+
+	set y(y: number) {
+        this._drawOffset.y = y;
+		this._adjustTextY();
+	}
 	//endregion
 
 	//region public methods
-	public updateSprites(serializedBlocks) {
-		for (let y = 0; y < this._height; y++) {
-			for (let x = 0; x < this._width; x++) {
-				if (serializedBlocks[y][x]) {
-					this._blockRows[y][x].sprite.setVisible(true);
-					this._blockRows[y][x].sprite.setFrame(serializedBlocks[y][x].spriteFrameName);
-				} else {
-					this._blockRows[y][x].sprite.setVisible(false);
-				}
-			}
-		}
+	public update(player: MatchPlayer) {
```

Comment:
I don't think this is a MatchPlayer; see the differences between MatchPlayer and Serialized(Match)Player in the interfaces

Reply:
I dont see any serialized players. Where exactly is the changed interface?

Reply:
Apparently, it's only [in the server code](https://github.com/TeamTetris/tetris-server/blob/1b0be662547914f9d426edbfd147aa377f4ec1b3/src/player/serializedMatchPlayer.ts).